packages/shared/src/components/CombatReport/CombatCurves/CurveChart.tsx

```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
+import { Legend, Line, LineChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
```

Comment:
Have you used recharts a lot? I've been using https://formidable.com/open-source/victory/docs/victory-chart/ in other repos to decent results

Reply:
yeah I've used recharts in other projects and it's pretty good. any particular feature in victory chart that you think would be useful for us? I can check if they are available through recharts.